Makefile

```diff
@@ -269,6 +269,37 @@ work/stunnel.conf:
 	@echo cafile=$(ROOT_DIR)/work/ca/certs/ca.cert.pem >> $@
 	@echo verify=2 >> $@
 
+	@echo [stunnel-master-slave-node-1] >> $@
```

Comment:
Having two tunnels should be sufficient

Reply:
✅ 2138906cc270e733be7a24d2544bd10e3d00d308